### PR TITLE
add a couple styles using custom variables

### PR DESCRIPTION
**Pull request by morethanreal**
@nevir PTAL cc @cdata 

I only added the few styles used by PolymerElements/paper-item but the idea is to add to it as needed. I'm not sure how to solve the duplication issue, since there's not a good way to include regular styles elsewhere.

### CODE REVIEW

**Comment by nevir** [comment]
Yeah... not sure how to deal with the duplication. I think we can get away with just transitioning everything to custom properties, though

File: bower.json [hunk 1]
```diff
@@ -17,5 +17,9 @@
   "ignore": [
     "/.*",
     "/demo/"
-  ]
+  ],
+  "dependencies": {
+    "font-roboto": "Polymer/font-roboto",
```
Comment (nevir) [inline bower.json]: Not actually using this

Comment (morethanreal) [inline bower.json]: Removed.

**Comment by morethanreal** [comment]
There's a cost to the custom properties shim, so I'm reluctant to not provide an option. But if elements are committing to using it, maybe it's fine?

**Comment by morethanreal** [comment]
@nevir Is it ok to merge this?

**Comment by nevir** [comment]
Sorry! yup!

**Pull request merged**